src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -707,6 +707,26 @@ private static FlowAnalysisAnnotations DecodeFlowAnalysisAttributes(PEModule mod
             return annotations;
         }
 
+        internal override ImmutableHashSet<string> NotNullIfParameterNotNull
+        {
+            get
+            {
+                // TODO2 cache?
+                var attributes = GetAttributes();
+                ImmutableHashSet<string>.Builder set = null;
```

Comment:
>ImmutableHashSet<string>.Builder set = null [](start = 16, length = 43)

Consider using `ImmutableHashSet<string>` directly rather than a `Builder`.